### PR TITLE
Make overall improvements in the home page

### PR DESCRIPTION
**Pull request by frozenhelium**
- Add CircularProgressBar component
- Create InformationCardFragments component
- Create InformationCard and PercentageInformationCard
- Remove InformationBox

### CODE REVIEW

File: src/components/ui/CircularProgressBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    const progressArc = (v: number) => arcGenerator({
+        endAngle: 2 * Math.PI * v,
+        innerRadius: arcInnerRadius,
+        outerRadius: arcOuterRadius,
+        startAngle: 0,
+    });
+
+    const unfilledPathData = progressArc(1);
+    const filledPathData = progressArc(value / 100);
```
Comment (tnagorra) [inline src/components/ui/CircularProgressBar/index.tsx]: Shouldn't some of these be memoized?

File: src/components/ui/InformationCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import Numeral from '#rscv/Numeral';
+import InformationCardFragments,
+{
```
Comment (tnagorra) [inline src/components/ui/InformationCard/index.tsx]:
```suggestion
import InformationCardFragments, {
```

File: src/components/ui/PercentageInformationCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import Numeral from '#rscv/Numeral';
+import CircularProgressBar from '#dui/CircularProgressBar';
+
+import InformationCardFragments,
+{
```
Comment (tnagorra) [inline src/components/ui/PercentageInformationCard/index.tsx]:
```suggestion
import InformationCardFragments, {
```

File: src/constants/styleProperties.js [hunk 4]
```diff
@@ -65,7 +65,7 @@ const styleProperties = {
 
         heightActionBar: '42px',
 
-        fontSizeExtremeLarge: '32px',
+        fontSizeUltraLarge: '38px',
```
Comment (tnagorra) [inline src/constants/styleProperties.js:69]: Sir. I would like to remind you that fontSizeExtremeLarge is used in 5 places.

File: src/views/Home/ProjectItem/index.tsx [hunk 5]
```diff
@@ -128,12 +133,12 @@ function ProjectItem(props: RecentProjectItemProps) {
             date: (new Date(pa.date)).getTime(),
         })).sort((a, b) => compareDate(a.date, b.date))
     ), [projectActivity]);
+    console.warn('time here', convertedProjectActivity, projectActivity);
```
Comment (AdityaKhatri) [inline src/views/Home/ProjectItem/index.tsx]: Remove plz

Comment (tnagorra) [inline src/views/Home/ProjectItem/index.tsx]: No console.warn meant for debugging

File: src/views/Home/RecentActivity/ActivityItem/index.tsx [hunk 6]
```diff
@@ -25,7 +25,7 @@ interface RecentActivityProps {
 function ActivityItem(props: RecentActivityProps) {
     const {
         createdAt,
-        createdByDisplayName,
+        createdByDisplayName = 'Unknown user',
```
Comment (tnagorra) [inline src/views/Home/RecentActivity/ActivityItem/index.tsx]: Don't we need to translate this?

File: src/components/ui/CircularProgressBar/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply src/components/ui/CircularProgressBar/index.tsx]: done

File: src/components/ui/InformationCard/index.tsx (hunk 2 above)
Comment (frozenhelium) [reply src/components/ui/InformationCard/index.tsx]: done

File: src/components/ui/PercentageInformationCard/index.tsx (hunk 3 above)
Comment (frozenhelium) [reply src/components/ui/PercentageInformationCard/index.tsx]: done

File: src/constants/styleProperties.js (hunk 4 above)
Comment (frozenhelium) [reply src/constants/styleProperties.js:69]: added it back

File: src/views/Home/RecentActivity/ActivityItem/index.tsx (hunk 6 above)
Comment (frozenhelium) [reply src/views/Home/RecentActivity/ActivityItem/index.tsx]: Removed the default value

**Pull request merged**